bigquery/storage/managedwriter/doc.go

```diff
@@ -24,12 +24,157 @@ feature-rich successor to the classic BigQuery streaming interface, which is pre
 in cloud.google.com/go/bigquery, and the tabledata.insertAll method if you're more familiar with the BigQuery v2 REST
 methods.
 
-Appending data is accomplished through the use of streams.  There are four stream types, each targetting slightly different
-use cases and needs.  See the StreamType documentation for more details about each stream type.
 
-This API uses serialized protocol buffer messages for appending data to streams.  For users who don't have predefined protocol
-buffer messages for sending data, the cloud.google.com/go/bigquery/storage/managedwriter/adapt subpackage includes functionality
-for defining protocol buffer messages dynamically using table schema information, which enables users to do things like using
-protojson to convert json text into a protocol buffer.
+Creating a Client
+
+To start working with this package, create a client:
+
+	ctx := context.Background()
+	client, err := managedwriter.NewClient(ctx, projectID)
+	if err != nil {
+		// TODO: Handle error.
+	}
+
+
+Defining the Protocol Buffer Schema
+
+The write functionality of BigQuery Storage requires data to be sent using encoded
+protocol buffer messages using proto2 syntax.  As the protocol buffer is not
```

Comment:
Nit: [Encoding / binary wire format](https://developers.google.com/protocol-buffers/docs/encoding) is the thing the API needs. I think of syntax as the language definition, which doesn't always apply given we can create dynamic messages in some languages.

```suggestion
protocol buffer messages using proto2 wire format.  As the protocol buffer is not
```

Reply:
Good callout.  I was considering "encoding" before, but was unhappy with it.  Wire format is cleaner.